docs/customization/index.rst

```diff
@@ -19,6 +19,7 @@ The Customization Guide is helpful while wanting to adapt Sylius to your persona
     state_machine
     grid
     tips_and_tricks
+    suites
```

Comment:
I'm wondering if `suites` page is meaningful enough. Maybe we should add `fixtures` here, and `suites` would be one of the sections?

Reply:
Let's only adjust `suites` naming and I think we should be fine
```suggestion
    fixture_suites
```